silk/migrations/0009_explicit_id_fields.py

```diff
@@ -0,0 +1,20 @@
+from django.db import migrations, models
+
+class Migration(migrations.Migration):
```

Comment:
I would argue this is a good opportunity to remove makemigrations from the readme.

Reply:
:+1:  :done: